src/util/timemanager.cpp

```diff
@@ -258,6 +258,10 @@ void TimeManager::progressTime(double dt) {
         // and time is not paused, just advance time.
         _deltaTime = _targetDeltaTime;
         _currentTime.data().advanceTime(dt * _deltaTime);
+        //Call the playback callback function if mode is enabled
+        if (_playbackModeEnabled) {
```

Comment:
Just replace this with `_playbackModeEnabled = false` (?) as the `if` doesn't do anything

Reply:
This if case used to have a callback. Changed as recommended.